DB/src/test/java/io/deephaven/util/calendar/StaticCalendarMethodsTest.java

```diff
@@ -24,6 +26,20 @@
     private final String date1 = "2017-08-01";
     private final String date2 = "2017-08-05";
 
+    private final DbCleanup base = new DbCleanup();
+
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+        base.setUp();
```

Comment:
Why do you need to do this in this class, but not in other ones?

Reply:
This is a junit3 test, I only intended to upgrade the ones that seemed already to be meant to be junit4